client/me/reauth-required/two-factor-actions.jsx

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+
+import { Button, Card } from '@automattic/components';
+import { localize } from 'i18n-calypso';
+import { recordTracksEventWithClientId as recordTracksEvent } from 'state/analytics/actions';
```

Comment:
Since there is a `recordTracksEvent`, we should be careful with aliasing something else to that name. We could just reference this as `this.props. recordTracksEventWithClientId` in the component and that would do the job.

Reply:
Adressed in d96147e3b119919a47c3ab4b09f5840b0b58fa20